package.json

```diff
@@ -14,7 +14,7 @@
   },
   "devDependencies": {
     "@types/jest": "^24.0.11",
-    "@types/node": "^11.13.5",
+    "@types/node": "^12.12.47",
```

Comment:
@thboop @ericsciple FYI I bumped this package version since all actions run on Node 12 and it was causing issues in the test, see https://github.com/actions/toolkit/pull/497#issuecomment-644996955